src/BaGet.Core.Server/Configuration/ConfigureForwardedHeadersOptions.cs

```diff
@@ -1,15 +1,31 @@
+using BaGet.Core;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.HttpOverrides;
 using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
 
 namespace BaGet.Configuration
 {
     public class ConfigureForwardedHeadersOptions : IConfigureOptions<ForwardedHeadersOptions>
     {
-        public void Configure(ForwardedHeadersOptions options)
+        private BaGetOptions bagetOptions;
```

Comment:
Please add the `readonly` field modifier to ensure no one accidentally modifies this value.
Also, please keep the options wrapped in a `IOptionsSnapshot` to support automatic configuration reload.

```suggestion
        private readonly IOptionsSnapshot<BaGetOptions> _options;
```

Reply:
I will update this also in next commit. I wasn't aware of the IOptionsSnapshot thing if I'm honest, but it makes sense.

Reply:
I couln't get it to work with `IOptionsSnapshot` because BaGetOptions and ConfigureForwardedHeadersOptions are in different scopes. Maybe you can get it to work.